<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="LAPTOP-D2GVCL39" ignored="0" name="Default test" tests="9" failures="1" timestamp="2022-08-02T20:01:02 IST" time="31.264" errors="0">
  <testcase name="verifyUserCanAddNewCategory" time="5.159" classname="regression.InventoryCategoryTest"/>
  <testcase name="verifyCategoryAddedSuccessMessageIsDisplayed" time="0.057" classname="regression.InventoryCategoryTest"/>
  <testcase name="verifyCategoryPageCountGetsIncreasedOnAddingNewCategory" time="3.24" classname="regression.InventoryCategoryTest"/>
  <testcase name="verifyUserCanEditExistingCategory" time="1.729" classname="regression.InventoryCategoryTest">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	Old category is still displayed on UI expected [false] but found [true]">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
Old category is still displayed on UI expected [false] but found [true]
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:47)
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:31)
at regression.InventoryCategoryTest.verifyUserCanEditExistingCategory(InventoryCategoryTest.java:56)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:132)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:599)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:174)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:822)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(Unknown Source)
at org.testng.TestRunner.privateRun(TestRunner.java:764)
at org.testng.TestRunner.run(TestRunner.java:585)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1218)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
at org.testng.TestNG.runSuites(TestNG.java:1069)
at org.testng.TestNG.run(TestNG.java:1037)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- verifyUserCanEditExistingCategory -->
  <testcase name="verifyCategoryUpdatedSuccessMessageIsDisplayed" time="0.056" classname="regression.InventoryCategoryTest"/>
  <testcase name="verifyCategoryPageCountWillBeSameAfterEdit" time="3.262" classname="regression.InventoryCategoryTest"/>
  <testcase name="verifyUserCanDeleteExistingCategory" time="6.354" classname="regression.InventoryCategoryTest"/>
  <testcase name="verifyCategoryDeletedSuccessMessageIsDisplayed" time="0.043" classname="regression.InventoryCategoryTest"/>
  <testcase name="verifyCategoryPageCountWillGetDecreasedAfterDelete" time="3.481" classname="regression.InventoryCategoryTest"/>
</testsuite> <!-- Default test -->
